packages/components/tests/dummy/app/templates/components/code-block.hbs

```diff
@@ -0,0 +1,419 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "CodeBlock Component"}}
+
+<Shw::Text::H1>CodeBlock</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Content</Shw::Text::H2>
+
+  <Shw::Flex @direction="column" {{style marginTop="32px"}} as |SF|>
+    <SF.Item>
+      <Shw::Flex @columns={{2}} {{style gap="2rem"}} as |SF2|>
+        <SF2.Item {{style width="calc(50% - 1rem)"}} @label="one line">
+          <Hds::CodeBlock @language="javascript" @value="console.log('I am JavaScript code');" />
+        </SF2.Item>
+
+        <SF2.Item {{style width="calc(50% - 1rem)"}} @label="multi-line">
+          <Hds::CodeBlock
+            @language="javascript"
+            @value="let codeLang='JavaScript';
+console.log(`I am ${codeLang} code`);"
+          />
+        </SF2.Item>
+      </Shw::Flex>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H3>Title and description</Shw::Text::H3>
+
+  <Shw::Flex @direction="column" {{style marginTop="32px"}} as |SF|>
+
+    <SF.Item>
+      <Shw::Flex @columns={{2}} {{style gap="2rem"}} as |SF2|>
+        <SF2.Item {{style width="calc(50% - 1rem)"}} @label="title">
+          <Hds::CodeBlock @value="console.log('I am JavaScript code');" @language="javascript" as |CB|>
+            <CB.Title>Title</CB.Title>
+          </Hds::CodeBlock>
+        </SF2.Item>
+
+        <SF2.Item {{style width="calc(50% - 1rem)"}} @label="description">
+          <Hds::CodeBlock @value="console.log('I am JavaScript code');" @language="javascript" as |CB|>
+            <CB.Description>Description</CB.Description>
+          </Hds::CodeBlock>
+        </SF2.Item>
+      </Shw::Flex>
+    </SF.Item>
+
+    <SF.Item>
+      <Shw::Flex @columns={{2}} {{style gap="2rem"}} as |SF2|>
+        <SF2.Item {{style width="calc(50% - 1rem)"}} @label="title and description">
+          <Hds::CodeBlock @value="console.log('I am JavaScript code');" @language="javascript" as |CB|>
+            <CB.Title>Title that may wrap on multiple lines if the parent container is limiting its width</CB.Title>
+            <CB.Description>
```

Comment:
Have you considered the possibility that consumers add a list here? (`<ul/ol>+<li>`). Maybe something to style too, just to be sure to cover the bases.

Reply:
No, we haven't. I've not seen such scenarios in our audits and I would be cautious to include them unless necessary as the description is intended to be brief.